app/queries/graphql/world_index_query.rb

```diff
@@ -0,0 +1,31 @@
+class Graphql::WorldIndexQuery
+  def initialize(base_path)
```

Comment:
**suggestion:** simplification

Is the base path necessary? Won't it always be the same for this (and likewise on other index pages)? If it's not necessary, it could probably be removed from the model's `.find` method too

Reply:
I'm guessing the theory is that the same controller could be used to render another identically laid out page, but with different content.  It seems highly unlikely though.

Will leave it to the team who own Collections whether they'd like to make this change.